### PR TITLE
Updated issue reporting in Hub

### PR DESCRIPTION
**Pull request by garimavs**
I have changed the content of the FAQ in the Hub with the link in the GTN.

### CODE REVIEW

**Comment by garimavs** [comment]
Hello, kindly look at this PR and suggest any improvements, if any.

**Comment by beatrizserrano** [comment]
@jennaj are you ok with this?

**Review by beatrizserrano — APPROVED** [review]
Thanks @garimavs! I think that this is ok, but would like to have @jennaj's approval before merging

**Comment by garimavs** [comment]
> Thanks @garimavs! I think that this is ok, but would like to have @jennaj's approval before merging

Sure!

File: content/issues/index.md
Comment (beatrizserrano) [inline content/issues/index.md:6]:
```suggestion
# [Galaxy Issue Reporting and Feature Requests](https://training.galaxyproject.org/training-material/faqs/galaxy/#reporting-usage-problems-security-issues-and-bugs) 
```
let's keep the titles so that the links are not broken

**Comment by garimavs** [comment]
Hey @beatrizserrano, I have updated the link titles!

**Comment by beatrizserrano** [comment]
thanks! let's merge this third one too :) We can make changes afterwards if @jennaj thinks we should

**Pull request merged**
